BackendProject/WebAPIs/PREMIUMAGENCY/Custom.cs

```diff
@@ -10,12 +10,22 @@ public class Custom
         public static string? setUserEventCustomPOST(byte[]? PostData, string? ContentType, string workpath, string eventId)
         {
             string? output = null;
+
+            LoggerAccessor.LogInfo($"setUserEventCustom POSTDATA: \n{PostData}");
+
             switch (eventId)
             {
                 case "95":
                     if (File.Exists($"{workpath}/eventController/MikuLiveEvent/setUserEventCustom.xml"))
                         output = File.ReadAllText($"{workpath}/eventController/MikuLiveEvent/setUserEventCustom.xml");
-                    LoggerAccessor.LogError($"[PREMIUMAGENCY] - SetUserEventCustom sent for MikuLiveEvent {eventId}!");
+                    LoggerAccessor.LogInfo($"[PREMIUMAGENCY] - SetUserEventCustom sent for MikuLiveEvent {eventId}!");
+
```

Comment:
This is a better fit between #DEBUG brackets

Reply:
What do you mean this part is normal Info like it should...